src/amo/components/LanguagePicker.scss

```diff
@@ -7,6 +7,8 @@ $selector-padding: 10px;
 }
 
 .LanguagePicker-selector {
+  -moz-appearance: none;
```

Comment:
Cool, I'll just enable it and include it in this patch. If you'd rather it separate just let me know and I'll file an issue/PR.